src/com/telesign/util/TeleSignRequest.java

```diff
@@ -497,4 +502,26 @@ private String encode(String data, String key)  throws java.security.SignatureEx
 
 		return result;
 	}
+	
+	/**
+	 * Set the TLS protocol to TLSv1.2
+	 */
+	private void setTLSProtocol() {

```

Comment:
I thought the idea was to let the user select from supported versions 1.1 and 1.2 and not force them to 1.2?